forms/uds/a1/form_a1_ivp_error_checks_mc.csv

```diff
@@ -0,0 +1,152 @@
+a1-ivp-m-100,100,Error,a1,I,LVLEDUC,Missingness,LVLEDUC must be present,LVLEDUC cannot be blank,Q10b. LVLVEDUC (level of education) cannot be blank,IF EDUCLVL=blank,,,Yes,No,
+a1-ivp-c-101,101,Error,a1,I,LVLEDUC,Conformity,LVLEDUC conforms,"LVLEDUC must be between 1 and 6, or 9",Q10b. LVLVEDUC (level of education) must be between 1 and 6 or 9,IF LVLEDUC<1 or (EDUC>6 and  is not 9),,,Yes,No,
+a1-ivp-m-102,102,Error,a1,I,MARISTAT,Missingness,MARISTAT must be present,MARISTAT cannot be blank,Q11. MARISTAT (marital status) cannot be blank,IF MARISTAT=blank,,,Yes,Yes,
+a1-ivp-c-103,103,,a1,I,MARISTAT,Conformity,MARISTAT conforms,MARISTAT must be between 1 and 6 or 9,"Q11. MARISTAT (marital status) must be between 1 and 6, or 9",IF MARISTAT<1 or (MARISTAT>6 and is not 9),,,Yes,Yes,
+a1-ivp-m-104,104,Error,a1,I,LIVSITUA,Missingness,LIVSITUA must be present,LIVSITUA cannot be blank,Q12. LIVSITUA (living situation) cannot be blank,LIVSITUA=blank,,,Yes,Yes,
+a1-ivp-c-105,105,Error,a1,I,LIVSITUA,Conformity,LIVSITUA conforms,"LIVSITUA must be between 1 and 6, or 9","Q12. LIVSITUA (living situation) must be between 1 and 6, or 9",IF LIVSITUA<1 or (LIVSITUA >6 and  is not 9),,,Yes,Yes,
+a1-ivp-m-106,106,Error,a1,I,RESIDENC,Missingness,RESIDENC must be present,RESIDENC cannot be blank,Q13. RESIDENC (type of residence) cannot be blank,IF RESIDENC=blank,,,Yes,Yes,
+a1-ivp-c-107,107,Error,a1,I,RESIDENC,Conformity,RESIDENC conforms,"RESIDENC must be between 1 and 4, or =6 or =9",Q13. RESIDENC (type of residence) must be between 1 and 4 or =6 or =9,IF RESIDENC<1 or (RESIDENC>6 and  is not 9),,,Yes,Yes,
+a1-ivp-c-108,108,Alert,a1,I,ZIP,Conformity,ZIP conforms,"ZIP must be between 006 and 999, or blank","Q14. ZIP (zip code) must be between 006 and 999, or blank",IF ZIP<006 or ZIP>999 ,,,Yes,Yes,
+a1-ivp-m-109,109,Error,a1,I,SERVED,Missingness,SERVED must be present,SERVED cannot be blank,Q15. SERVED (served in armed forces) cannot be blank,IF SERVED=blank,,,Yes,No,
+a1-ivp-c-110,110,Error,a1,I,SERVED,Conformity,SERVED conforms,"SERVED must be 0, 1, or 9","Q15. SERVED (served in armed forces) must be 0, 1, or 9","IF SERVED is not 0,1,9",,,Yes,No,
+a1-ivp-m-111,111,Error,a1,I,MEDVA,Missingness,MEDVA conditionally present,If SERVED is 1 then MEDVA cannot be blank,If Q15. SERVED (served in armed forces) is 1 then Q16. MEDVA (VA medical care) cannot be blank,IF SERVED=1 and MEDVA=blank,,,Yes,No,
+a1-ivp-m-112,112,Error,a1,I,MEDVA,Missingness,MEDVA conditionally blank,If SERVED is 0 then MEDVA must be blank,If Q15. SERVED (served in armed forces) is 0 then Q16. MEDVA (VA medical care) must be blank,IF SERVED=0 and MEDVA is not blank,,,Yes,No,
+a1-ivp-c-113,113,Error,a1,I,MEDVA,Conformity,MEDVA conforms,"If SERVED is 1 then MEDVA must be 0, 1, or 9","If Q15. SERVED (served in armed forces) is 1 then Q16. MEDVA (VA medical care) must be 0, 1 or 9","IF SERVED=1 and (MEDVA is not 0,1,9)",,,Yes,No,

```

Comment:
hmm I think they can still technically answer MEDVA since they may not remember whether they served in the military but it may be clear through medical records or more recent memory that they had VA care. Aileen said maybe we should write an alert for when MEDVA= 1 (Yes) and SERVED=9 (Unknown) because SERVED should really be equal to 1 in that case. but it may happen so it's better as an alert vs an
 error. I added this as a new edited version of error a1-ivp-p-1006